lib/queue/consumer.ts

```diff
@@ -131,11 +138,21 @@ export class Consumer implements QueueConsumer {
 				pgPool: this.pool,
 				concurrency: defaultEnvironment.queue.concurrency,
 				pollInterval: 1000,
-				logger: new Logger((_scope) => {
-					return _.noop;
-				}),
+				logger: new Logger(graphileLoggerToJellyfishLogger(logContext)),
 				taskList: {
-					actionRequest: async (result) => {
+					actionRequest: async (result, helpers) => {
+						const eventLogData: ActionLogData = helpers.job
+							.payload as ActionLogData;
+
+						helpers.logger.info(
```

Comment:
I'm guessing this is temporary and just here to help with development?

Reply:
Mainly it is. We could also log what is being consumed ( similar to what the producer does ).
I'll clean it up anyway

Reply:
Cleaned it up and moved to `debug` category.